OWNERS

```diff
@@ -1,27 +1,20 @@
 approvers:
 - justinsb
-- zmerlynn
-- gnufied
-- jsafrane
 - nckturner
-- micahhausler
-- mcrute
```

Comment:
cc @mcrute https://github.com/kubernetes/cloud-provider-aws/commits?author=mcrute

Reply:
I haven't had much time to dedicate to this lately so I'm okay with being removed as an active maintainer. Any reason not to just push me into emeritus instead?

Reply:
You're in emeritus, see line 18 (diff is rendered weirdly).

Reply:
Thanks for the cleanup! I read the diff out of context. I hope to have more time to contribute to cloud provider in the future. Until then...

/lgtm